packages/openid4vc/src/openid4vc-holder/__tests__/fixtures.ts

```diff
@@ -39,6 +42,8 @@ export const matrrLaunchpadDraft11JwtVcJson = {
       {
         id: 'b4c4cdf5-ccc9-4945-8c19-9334558653b2',
         format: 'ldp_vc',
+        // FIXME: should we make this optional? Seems a lot of types break on this
```

Comment:
```suggestion
        // TODO: should we make this optional? Seems a lot of types break on this
```

Do we differentiate between FIXME and TODO? We should keep it conistent if they mean the same.

Might also be nice to have something like `// TODO(oid): bla bla`

Reply:
They are different. I fixed quite a few and renamed some to todo